metadata/src/main/java/org/apache/kafka/controller/util/SnapshotFileReader.java

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-package org.apache.kafka.shell;
+package org.apache.kafka.controller.util;
```

Comment:
Hmm, I don't think this is really controller-specific, is it? In fact, we don't use it in the controller per se (although it is used in the tooling...)

Perhaps `org.apache.kafka.metadata.util` ?